packages/extension/src/shared/token/storage.ts

```diff
@@ -51,3 +60,50 @@ export async function removeToken(token: BaseToken) {
   await assertSchema(baseTokenSchema, token)
   return tokenStore.remove((t) => equalToken(t, token))
 }
+
+export async function updateTokenList() {
```

Comment:
If it's a manual update trigger by user's click, we should do an update immediately.

Maybe the function should look like this:
```
export async function updateTokenList(forced?: boolean) {
  if (forced? or didNotUpdateForOneHour) {
    update()
  } else {
    console.debug(`Update skipped`)
  }
}
```
Then we call forced update when it is a manual update.

Reply:
If it is manual update, then `updateTokenListNow` will be triggered:
https://github.com/alephium/extension-wallet/blob/auto-update-token-list/packages/extension/src/ui/features/settings/DeveloperSettings.tsx#L46

Reply:
I see. That's a much better approach 👍